src/common/options.cc

```diff
@@ -7291,6 +7291,11 @@ std::vector<Option> get_mds_options() {
     Option("mds_max_retries_on_remount_failure", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
      .set_default(5)
      .set_description("number of consecutive failed remount attempts for invalidating kernel dcache after which client would abort."),
+
+    Option("mds_dump_cache_threshold", Option::TYPE_SIZE, Option::LEVEL_DEV)
+     .set_default(1LL<<30)
```

Comment:
Let's use `1_G` instead to ease readability.

Reply:
done